ghost.py

```diff
@@ -162,106 +174,106 @@ def get(self, key, decrypt=True):
     def delete(self, key):
         """Delete a key if it exists.
         """
-        record = self._get(key)
```

Comment:
Note that this is a race condition really... the "get and delete" needs to be on the Storage side, to allow for using backend-specific atomic operations (so with sql, the result of a `DELETE` query tells you if you actually deleted anything or not)